custom-addons/esb/packing_list/models/packing_list.py

```diff
@@ -120,7 +120,7 @@ class ESBPackingList(models.Model):
     picking_ids = fields.Many2many(
         comodel_name="stock.picking",
         string="Packing List",
-        domain="[('partner_id', '=', partner_id)]",
+        domain="[('partner_id','=',partner_id),('location_dest_id', '=', 5)] ",
```

Comment:
Should you filter by `picking_type_id`? I think it better than filter by `location_dest_id`.